runtime/sys.c

```diff
@@ -443,11 +449,12 @@ CAMLprim value caml_sys_modify_argv(value new_argv)
 
 CAMLprim value caml_sys_executable_name(value unit)
 {
-  return caml_copy_string_of_os(caml_exe_name);
+  return caml_copy_string_of_os(caml_params->exe_name);
 }
 
 void caml_sys_init(char_os * exe_name, char_os **argv)
 {
+  value v;
```

Comment:
Not necessary. Remove.

Reply:
Fixed in https://github.com/ocaml-multicore/ocaml/commit/9ef07afa0adc5b5d764c2a1e59c9551bb6a0e5f6